integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -109,25 +109,48 @@ public static void main(String[] args) {
           + "Proceed with local configuration for FUSE: {}", e.toString());
     }
     conf = fsContext.getClusterConf();
-    final AlluxioFuseOptions opts = parseOptions(args, conf);
+    final FuseMountInfo opts = parseOptions(args, conf);
     if (opts == null) {
       System.exit(1);
     }
+    try {
+      launchFuse(fsContext, opts, true);
```

Comment:
return value not used? Then do we need a return a `FuseUnmountable ` for this method?
And also do we even need `FuseUnmountable` since it is not used?

Reply:
Remove the `FuseUnmountable` because it's not needed anymore since we decide to remove the support for runtime mount/unmount